addons/mail/static/src/messaging/component/notification_group/notification_group.scss

```diff
@@ -0,0 +1,107 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_NotificationGroup {
+    @include o-mail-notification-list-item-layout();
+
+    &:hover .o_NotificationGroup_markAsRead {
+        // TODO SEB also mixin this
+        opacity: 1;
```

Comment:
again, copy from thread preview...

Reply:
not a reason not to put in correct category here, no?

Reply:
I prefer to keep them consistent with each other so that they are easier to merge in task-2258605.
And I don't want to start changing those that I didn't modify in the current PR.